resources/views/site/home.blade.php

```diff
@@ -69,14 +73,46 @@
 							<table class="table table-sm table-borderless pb-0 mb-0">
 								<tr class="p-3 m-0">
 									<td class="py-0 pl-1 pr-2 m-0">
-										@if($atl_ctr === 1)
+										@if($center === 1)
 										<span class="badge bg-success">ONLINE</span>
 										@else
 										<span class="badge bg-danger">OFFLINE</span>
 										@endif
 									</td>
 									<td class="py-0 px-2 m-0 text-white">Atlanta Center</td>
 								</tr>
+
+								@foreach ($fields as $field)
+									<tr class="p-3 m-0">
+										<td class="py-0 pl-1 pr-2 m-0">
+											@if($field['online'] === 1)
+												<span class="badge bg-success">ONLINE</span>
+											@else
+												<span class="badge bg-danger">OFFLINE</span>
+											@endif
+										</td>
+										<td class="py-0 px-2 m-0 text-white">
+											{{ $field['name'] }} ATCT
+
+											@foreach ($field['subfields'] as $subfield)
+												<span class="badge {{ $subfield['color'] }}">{{ $subfield['short'] }}</span>
+											@endforeach
+										</td>
+									</tr>
+
+									<tr class="p-3 m-0">
+										<td class="py-0 pl-1 pr-2 m-0">
+											@if($field['approach']['online'] === 1)
+												<span class="badge bg-success">ONLINE</span>
+											@else
+												<span class="badge bg-danger">OFFLINE</span>
+											@endif
+										</td>
+										<td class="py-0 px-2 m-0 text-white">{{ $field['name'] }} Approach</td>
+									</tr>
+								@endforeach
+
+								{{--
```

Comment:
Let's avoid commenting things out to keep the code cleaner. If something needs to be commented out, remove it. If we need it again in the future, we can grab it from a past version using git. That's the beauty of version control